client/marklogicUnitTestClient.ts

```diff
@@ -0,0 +1,85 @@
+ */
+
+'use strict';
+
+import {
+    ExtensionContext, TextEditor, WorkspaceConfiguration
+} from 'vscode';
+
+import { ClientContext, MlClientParameters, requestMarkLogicUnitTest } from './marklogicClient';
+import { buildMlxprsErrorFromError, MlxprsError } from './mlxprsErrorBuilder';
+import { MlxprsErrorReporter } from './mlxprsErrorReporter';
+import { MlxprsWebViewProvider } from './mlxprsWebViewProvider';
+
+export class MarkLogicUnitTestClient {
+    static mlxprsWebViewProvider: MlxprsWebViewProvider = null;
+
+    public static registerMlxprsResultsViewProvider(mlxprsWebViewProvider: MlxprsWebViewProvider) {
+        MarkLogicUnitTestClient.mlxprsWebViewProvider = mlxprsWebViewProvider;
+    }
+
+    private extensionContext: ExtensionContext;
+
+    public constructor(context: ExtensionContext) {
+        this.extensionContext = context;
+    }
+
+    private newMarklogicUnitTestClient(cfg: WorkspaceConfiguration): ClientContext {
+        const configParams: Record<string, unknown> = {
+            host: String(cfg.get('marklogic.host')),
+            user: String(cfg.get('marklogic.username')),
+            pwd: String(cfg.get('marklogic.password')),
+            port: Number(cfg.get('marklogic.unitTestPort')),
+            contentDb: null,
+            modulesDb: null,
+            authType: String(cfg.get('marklogic.authType')).toUpperCase(),
+            ssl: Boolean(cfg.get('marklogic.ssl')),
+            pathToCa: String(cfg.get('marklogic.pathToCa') || ''),
+            rejectUnauthorized: Boolean(cfg.get('marklogic.rejectUnauthorized'))
+        };
+        const newParams = new MlClientParameters(configParams);
+        return new ClientContext(newParams);
+    }
+
+    public async sendTestFile(
+        cfg: WorkspaceConfiguration, editor: TextEditor
+    ): Promise<void> {
+        const splitString = 'test/suites/';
+        const filePath = editor.document.uri.path;
+        const splitLocation = filePath.lastIndexOf(splitString) + splitString.length;
+        if (splitLocation > splitString.length) {
+            const testPath = filePath.substring(splitLocation);
+            const lastSlash = testPath.lastIndexOf('/');
+            const testSuite = testPath.substring(0, lastSlash);
+            const testFile = testPath.substring(lastSlash + 1);
+
+            const dbClientContext: ClientContext = this.newMarklogicUnitTestClient(cfg);
+            requestMarkLogicUnitTest(dbClientContext, testSuite, testFile)
+                .result((testResults: string) => {
+                    const testResultsHtml = MlxprsWebViewProvider.convertXmlResponseToHtml(testResults);
+                    MarkLogicUnitTestClient.mlxprsWebViewProvider.updateViewContent(testResultsHtml);
+                    return null;
+                })
+                .catch(error => {
+                    const mlxprsError: MlxprsError = buildMlxprsErrorFromError(error, `Unable to run the test(s): ${error['code']}`);
```

Comment:
Change the message to "Unable to run the test module" as part of standardizing on "run test module" or "run marklogic-unit-test module".